src/help/zaphelp/contents/releases/2_5_0.html

```diff
@@ -15,36 +15,31 @@
 
 <H2>Enhancements:</H2>
 <ul>
-<li>Issue 69 : Ability to export request into various executable forms</li>
-<li>Issue 222 : Report : allow to include details of request</li>
-<li>Issue 266 : Add auto tagging for setting cookies, json, anything else?</li>
-<li>Issue 295 : Grouping</li>
-<li>Issue 304 : fuzzer enhancements (was output should have checks)</li>
-<li>Issue 322 : Official Apt or Yum repositories</li>
-<li>Issue 396 : Addition functionality - possibility to follow up request in Resend Dialog</li>
-<li>Issue 464 : Split the RFI tests out of PathTraversal</li>
-<li>Issue 605 : Force intercepts via header</li>
+<li>Issue 266 : Add auto tagging for setting cookies, json</li>
 <li>Issue 646 : Password outgoing proxy in plain text on screen</li>
-<li>Issue 937 : Move copy and past extensions into stdmenus</li>
-<li>Issue 1015 : Support Server Name Indication</li>
+<li>Issue 1171 : Sites tree to raise 'new site / new node' events</li>
 <li>Issue 1229 : Update Extensions/Plugins to Take Advantage of New Confidence Scale</li>
 <li>Issue 1341 : Update alert XML schema to remove element-level name collision</li>
-<li>Issue 1496 : Nit: Unify Mac OS X "About Box" with Java "About Box"</li>
 <li>Issue 1590 : Mode to apply when running as a daemon</li>
 <li>Issue 1713 : Source Code Disclosure SVN Throws False Positive</li>
+<li>Issue 1864 : Copy scan progress as text to clipboard</li>
 <li>Issue 1958 : Allow to disable database (HSQLDB) log</li>
 <li>Issue 1959 : Allow to active scan headers of all requests</li>
 <li>Issue 1980 : Add ZAP CLI to Docker images</li>
 <li>Issue 2068 :  LoggedIn and LoggedOut indicators inside authentication scripts</li>
 <li>Issue 2070 : It should be possible for the authentication scripts to configure how the messages are sent</li>
 <li>Issue 2094 : Support a '-addoninstallall' command line option</li>
 <li>Issue 2121 : Disable Req/Resp tabs position options when in "Expand Full" layout</li>
+<li>Issue 2127 : Warn if the report generation failed</li>
 <li>Issue 2134 : Update HSQLDB library to version 2.3.3</li>
```

Comment:
This entry can be removed, superseded by issue 2512 - Update HSQLDB library to version 2.3.4.